### ISSUE TITLE
Results - baseline, actual and target comment fields are not displayed

### ISSUE THREAD

**Issue by stevieflow** [devinit/D-Portal#609]
The IATI standard enables users to include narratives for the baseline, target and actual elements

Here's an example: http://d-portal.org/q.xml?aid=BE-BCE_KBO-0478060441-PROG2017-2021_OSS4

```
<baseline year="2016" value="0">
<comment>
<narrative>- SACCAWU organised in 211 companies/ company groups, within the hospitality sector covering around 320.000 workers, from which 70% are non-STD employment</narrative>
</comment>
</baseline>
<period>
<period-start iso-date="2017-01-01"/>
<period-end iso-date="2019-12-31"/>
<target value="1">
<comment>
<narrative>By 2019 CBA’s frames of SACCAWU are adjusted, complying to the new Labour Relationship Act legislation, benefiting vulnerable workers from the hospitality sector, from which 35% are female and 15% younger than 35 years.</narrative>
</comment>
</target>
<actual value="1">
<comment>
<narrative xml:lang="en">The Organising, Campaigns and Collective Bargaining Unit (OCCBU) of SACCAWU have held workshops and meetings to train 330 trade union officials to adjust the CBA’s with (a) the new legislation concerning the national minimum wage (NMW) and (b) a safe late night shift transport/allowance, benefiting all the 17.815 members concerning the NMW and giving a late night shift transport allowance to those members under the 20 CBA re-negotiated in the hospitality sector, from which 39% female and 16% younger than 35y.</narrative>
</comment>
</actual>
```

However, it doesnt appear that d-portal displays these: http://d-portal.org/q.html?aid=BE-BCE_KBO-0478060441-PROG2017-2021_OSS4

![d-portal-org-q-html-aid-BE-BCE_KBO-0478060441-PROG2017-2021_OSS4](https://user-images.githubusercontent.com/1209944/109686035-cddcf000-7b79-11eb-82a6-ac339fab96fc.png)

These narratives will be important clarification points on results  - so it'd be good to include them, somehow,,,,,

For info, a fair number of publishers use this, too - here's the narrative on actuals:  http://dashboard.iatistandard.org/element/iati-activity_result_indicator_period_actual_comment.html

**Comment by notshi** [comment]
Thanks, @stevieflow - we are aware of this.

The new SAVi update (part of the overhaul new d-portal architecture and query backend) included the redesign of SAVi, including the Results section where we were aiming for an easier view at the rather complex nature of results data visualisation.

We will need some design/dev time to consider where and how best to display the comment fields, while keeping the simplified layout as it is now.

**Comment by notshi** [comment]
Hi @stevieflow 

> NOTE: a side issue on this is that d-portal does not currently display the written comment elements of any result/indicator. This might be important context if @publishwhatyoufund reviewers are using d-portal (which is very helpful in many many other aspects): devinit/D-Portal#609

Thanks for bringing the 2022 Aid Transparency Index to our attention.
Comments in result data should now be displayed in SAVi.

Hopefully the change isn't too jarring when you're trying to read the results, though it depends on the published data.
I've had to move each comment to their own row as they can get very lengthy(!).
It was quite a challenge trying to make the comments look like they are connected to a certain period / value so tabs were what we ended up with after many iterations over permutations of data.

For example, here's one on the extreme side, where the publisher has added a tonne of white space to their comments.
json  - http://d-portal.org/q.json?&from=xson&root=/iati-activities/iati-activity&aid=XM-DAC-41114-OUTPUT-00102280
SAVi - http://d-portal.org/ctrack.html#view=act&aid=XM-DAC-41114-OUTPUT-00102280

Here's one with a good mixture of dimensions and comments.
http://d-portal.org/ctrack.html#view=act&aid=US-EIN-13-5660870-EN017
